packages/jellyfish-api-core/src/category/wallet.ts

```diff
@@ -412,3 +423,43 @@ export interface WalletFlagResult {
   flag_state: boolean
   warnings: string
 }
+
+export interface InWalletTransaction {
+  amount: number
+  fee?: number
```

Comment:
Should remove ? like line 448

Reply:
I'm not exactly sure I should.

Some of the `txid` that I've tested actually has some of the keys missing. Therefore I'm not exactly sure if they are mandatory

https://github.com/DeFiCh/jellyfish/pull/342#discussion_r645383891

Reply:
Let say you are able to test a scenario where the fee is removed and another scenario where there is a fee. Then you can put '?'

Else should remove '?'